### PR TITLE
Add environment.yml to make installation easier

### PR DESCRIPTION
**Pull request by filipre**
Hi, 
to make the installation easier (for me), I added all dependencies into one file using `conda`. Feel free to merge.
Cheers
René

### CODE REVIEW

**Comment by Kaixhin** [comment]
I wasn't aware of this feature of `conda`, so thanks for bringing it to my attention! I won't merge this but I'll add the feature in myself. Adding some comments to your PR anyway.

File: .gitignore [hunk 1]
```diff
@@ -93,3 +93,6 @@ ENV/
 *.pth
 # Plots
 *.html
+
+# OSX related
```
Comment (Kaixhin) [inline .gitignore:97]: [Global .gitignores](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore) are the way to go here.

File: environment.yml [hunk 2]
```diff
@@ -0,0 +1,92 @@
+name: rainbow
+channels:
+- pytorch
+- defaults
+dependencies:
+- asn1crypto=0.23.0=py36h782d450_0
```
Comment (Kaixhin) [inline environment.yml:6]: These versions look rather specific, and are definitely Python 3.6, so these can't be general enough for most setups.

File: .gitignore (hunk 1 above)
Comment (filipre) [reply .gitignore:97]: ahh, I wasn't aware of *that* feature as well. Thanks!

**Pull request closed without merging**